README.rst

```diff
@@ -1,40 +1,49 @@
 Plugin Template
 ===============
-This is a template project that demonstrates how one can create and upload a manim 
-plugin to PyPI using a PEP 517 compliant build system (in this case
-`poetry <https://python-poetry.org>`_). This build system ensures users of
-your plugin are able to do so reliably without falling into dependency hell.
-You may use another build system other than poetry (e.g. Flit, Enscons,
-etc...) if you wish to.
+Plugins are features that extend Manim's core functionality. This is a
+template project that demonstrates how you can create and upload a manim
+plugin to PyPI using a PEP 517 compliant build system, `Poetry
+<https://python-poetry.org>`_.
+
+Poetry is **NOT** required to create plugins, but is recommended because it
+provides build isolation and ensures users can reliably install your plugin
+without falling into dependency hell. You may use another build system (e.g.
+Flit, Setuptools, Pipenv, etc...) if you wish. 
 
 Creating Plugins
 ----------------
-Plugins are features that can be used to extend the core of Manim's
-functionality. By convention, plugins should be prefixed by ``manim-`` when
-being named. This allows users to easily search for plugins on organizations
-like PyPi, but it is not required. Manim discovers user plugins via the
-metadata specified in the ``pyproject.toml``.
+The only requirement of your preferred build system is that it specifies the
+``manim.plugins`` `entry point
+<https://packaging.python.org/specifications/entry-points/>`_.
+
+.. note:: 
```

Comment:
I don't think github/PyPi supports this syntax?

Reply:
Do you have any such recommendations? (I believe you're referring to the .. note:: syntax). I might convert from .rst to .md, or install a dev-dependency for the package to check syntax against PyPi.

Reply:
Just remove this `.. note::` like that of sphinx, instead use Headings or somethings. Try to edit in Github Web Editor there is even live preview feature and AFAIK, PyPi supports Github like Syntax.

Reply:
It seems like it's supported, but I can still change it. I uploaded to repository.testpypi:
https://test.pypi.org/project/manim-plugintemplate/

Reply:
Ok then seems good.